.github/workflows/publish.yml

```diff
@@ -0,0 +1,18 @@
+name: Build & publish module to registry 
+
+on:
+  push:
+    tags:
+    - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
add two lines here, `- '[0-9]+.[0-9]+.[0-9]+-rc'` and `- '[0-9]+.[0-9]+.[0-9]+-rc[0-9]+' so that release candidates also get published to the repository